frontend/src/pages/RequestDetailsPage/v2/otel-helpers.ts

```diff
@@ -90,6 +90,10 @@ export function getNumber<T = number>(
     defaultValue: T;
   },
 ) {
+  if (typeof value === "string") {
```

Comment:
nit: Lau and discovered today that `Number("")` returns 0. i'm going to add a case that this is not an emptystring. then, approved!